macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNLocation/VPNLocationViewModel.swift

```diff
@@ -93,10 +94,14 @@ final class VPNLocationViewModel: ObservableObject {
 
     @MainActor
     private func reloadList() async {
-        guard let locations = try? await locationListRepository.fetchLocationList().sortedByName() else { return }
+        guard let locations = (try? await locationListRepository.fetchLocationList().sortedByName()) ?? Self.cachedLocations else {
```

Comment:
This line was the cause of the unresponsive UI. When the locations endpoint is unreachable, this hits an early return, so the list of cities doesn't get properly refreshed. After the change, it will try to refresh, but fall back to the last fetched values otherwise.